### PR TITLE
Grant Detail Page Redesing

### PR DESCRIPTION
**Pull request by corydickson**
##### Description

This implements the grant page redesign from the figma. What this doesn't include is the graph which functionality still needs to be determined. And adding a comments to a particular transaction.

Furthermore, in the related grants sections, I've kept the team members icons as if there are more than a single member team member it's not clear which user should be below the byline.

### CODE REVIEW

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
@corydickson Could you please add screenshot / recordings

File: app/assets/v2/js/grants/_detail-component.js [hunk 1]
```diff
@@ -11,6 +11,63 @@ Quill.register('modules/ImageExtend', ImageExtend);
...
+    fetchTransactions: function() {
+      let vm = this;
+
+      page = vm.transactions.next_page_number;
+      if (!page) {
+        return;
+      }
+      vm.loadingTx = true;
+
+      let url = `/grants/v1/api/grant/${vm.grant.id}/contributions?page=${page}`;
```
Comment (thelostone-mc) [inline app/assets/v2/js/grants/_detail-component.js:54]: why not const ?

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
- RSK cross chain blockchain link are missing 
- latest contribution looks broken [SEE IMAGE](https://user-images.githubusercontent.com/5358146/109512454-205ed380-7aca-11eb-93eb-82b228d51af3.png)
- on editing a grant -> all the other payout_address get set to null and UI looks weird . [SEE IMAGE](https://user-images.githubusercontent.com/5358146/109511929-931b7f00-7ac9-11eb-9536-74e87cd69731.png)
- [logo overlaps text box. Also better placeholder. ?](https://user-images.githubusercontent.com/5358146/109512196-d970de00-7ac9-11eb-9633-64ce526e7ffc.png)
- Region is missing on the UI
- Can we hide related grants if it doesn't exist on a grant ?
- Can we hide show more when all related grants are all shown ? Doesn't make sense to show it else.
[ See Image](https://user-images.githubusercontent.com/5358146/109512821-7e8bb680-7aca-11eb-99ea-b285c23ba8a7.png)

 

image



This PR does not look ready

**Comment by octavioamu** [comment]
Closing as implemented on #8500. thanks @corydickson

**Pull request closed without merging**